### DISCUSSION TITLE
no line graphs and restricted to 300 rows

### DISCUSSION THREAD

**Discussion by vlsireddy** [microsoft/SandDance#556] [General]
Hi,

brief:
sql server <--> azure data studio {sanddance}

questions:
1. I see the rowcnt restriction to 300. How to set it to more?
2. What is the max row count, which it can handle ?
3. can the plots be put on a web server {http::/xyz:port}
4. I dont see there is line plot option, other xls type chart options. 

Thanks

**Comment by danmarshall** [comment]
Hello,
1. Is your query/data limited? I'm using Potgres and able to get thousands.
2. No limit, but after about half a million, you may see performance degrade
3. Not sure what you mean. Such infrastructure could be a security leak of your data?
4. Currently we don't have line chart since we show each row as "units" - A line would be "between units" and there would not be a visual transition between chart types.
